Packs/Cylance_Protect/Playbooks/playbook-Endpoint_Enrichment_-_Cylance_Protect_v2.yml

```diff
@@ -84,19 +88,23 @@ tasks:
     note: false
     timertriggers: []
     ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
   "2":
     id: "2"
-    taskid: 6fc275f9-cd51-4d40-8fe4-52a92bbc6af7
+    taskid: 79a6eab0-7d4b-49d2-8348-7abcfbec8bb5
     type: regular
     task:
-      id: 6fc275f9-cd51-4d40-8fe4-52a92bbc6af7
+      id: 79a6eab0-7d4b-49d2-8348-7abcfbec8bb5
       version: -1
       name: Get all Cylance Protect devices
-      description: Gets information about all devices that are available in Cylance Protect.
-      script: Cylance Protect v2|||cylance-protect-get-devices
+      description: Allows a caller to request a page with a list of Console device
```

Comment:
What was the change here? seems like a new description

Reply:
Fiexed.